conf/logback.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="net.logstash.logback.encoder.LogstashEncoder" />
+    </appender>
+
+    <appender name="ASYNCSTDOUT" class="ch.qos.logback.classic.AsyncAppender">
+        <appender-ref ref="STDOUT" />
+    </appender>
+
+    <logger name="com.gu.contentapi.sanity" level="INFO"/>
```

Comment:
Can / should `logger name` be `com.gu.contentapi.sanity-tests`?

Reply:
Well it refers to the java class path within the app... which is `com.gu.contentapi.sanity.{classname}` not `sanity-tests`. The purpose of this line is to capture `INFO` level logs from our code, but `WARN` level logs from everything else.